src/app/settings/WishListSettings.tsx

```diff
@@ -61,6 +60,20 @@ class WishListSettings extends React.Component<Props> {
             <div className="setting">
               <FileUpload onDrop={this.loadWishList} title={t('WishListRoll.Import')} />
             </div>
+            <div className="setting">
+              <div>{t('WishListRoll.ExternalSource')}</div>
+              <div>
+                <input
+                  type="text"
+                  className="wish-list-text"
+                  onChange={_.noop}
+                  onInput={this.wishListSourceChangeEvent}
```

Comment:
I believe this will fetch a new wishlist on every keystroke. Better to bind this to `onBlur` or an explicit button.

Reply:
Yeah, it does do it on every keystroke. It was acting a little funky, so a blur/button would be a QoL improvement.